core/domain/param_domain_test.py

```diff
@@ -21,9 +21,31 @@
 import utils
 
 
-class ParameterDomainUnitTests(test_utils.GenericTestBase):
+class ParamSpecUnitTests(test_utils.GenericTestBase):
     """Tests for parameter domain objects."""
+    # Unittest to test to_dict function.
+    def test_to_dict(self):
+        initial_vals = {
+            'obj_type': 'UnicodeString',
+        }
+        param_spec_obj = param_domain.ParamSpec('UnicodeString')
+        # ParamSpec.from_dict(initial_vals)
+        self.assertDictEqual(
+            initial_vals,
+            param_spec_obj.to_dict())
 
+    # Unittest to test from_dict function.
+    def test_from_dict(self):
+        initial_vals = {
+            'obj_type': 'UnicodeString',
+        }
+        param_spec_obj = param_domain.ParamSpec('UnicodeString')
+        obj_from_dict = param_spec_obj.from_dict(initial_vals)
+        self.assertDictEqual(
+            param_spec_obj.to_dict(),
+            obj_from_dict.to_dict())
+
+    # UnitTest to test validate function in ParamSpec class.
```

Comment:
Move the comment inside function.

Ditto elsewhere.

Reply:
Done